src/journal/journal-file.c

```diff
@@ -3247,10 +3250,20 @@ int journal_file_open(
 #elif HAVE_XZ
         f->compress_xz = compress;
 #endif
+
+        if (compress_threshold_bytes == (uint64_t) -1)
```

Comment:
not that it matters, but you opted to pass UINT64_MAX as parameters when calling the function and you check for (uint64_t) -1 here. This is of course the same, but still a bit strange to write it as one on the "sending" side, and as the other on the "receiving" side…

Reply:
I just switched everything to (uint64_t) -1 for consistencies sake. I prefer that to UINT64_MAX because it's the standard the JournalMetrics struct (for example) sets.